streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorTopology.java

```diff
@@ -94,6 +92,15 @@ boolean isRepartitionTopic(final String topic) {
         return repartitionTopics.contains(topic);
     }
 
+    boolean hasStateWithLoggingEnabled() {
```

Comment:
This is a bit misleading, since it will (and should) return true for optimized source KTables, which are technically marked as `loggingDisabled` -- as always, improved naming suggestions are welcome